packages/utils/test/formatingSpace.test.ts

```diff
@@ -0,0 +1,303 @@
+import { expect } from 'chai';
+import { removeFormattingSpace } from '../src/formattingSpace';
+
+/**
+ * @param {String} html representing any number of sibling elements
+ * @return {NodeList}
+ */
+const htmlToElements = (html: string, debug = false): NodeList => {
+    if (debug) console.log('html String = ', html);
+    const template = document.createElement('template');
+    template.innerHTML = html;
+    return template.content.childNodes;
+};
+
+describe('removeFormattingSpace() util function', () => {
+    describe('only child node', () => {
+        it('(01) Should return trimmed text when no space are around text', async () => {
```

Comment:
"when no space around text"
Also another very small detail but this doesn't follow the same nomenclature as our other tests (`it('should ...')` whereas you wrote `it('(##) Should ...')`)

Reply:
Yeah, I can remove the numbers if you want. I find it easier to find failed tests with the numbering but it's not a deal breaker for me.

Your call

Reply:
Well I do like consistency and I like to think if we wanted them to be numbered we could have that automatized so I'd say yeah please :-D